tools/sanity_checks.py

```diff
@@ -305,6 +305,8 @@ def check_source_url(self, name: str, wrap_section: configparser.SectionProxy, v
         elif name == 'netstring-c':
             # There is no specific version for netstring-c
             return True
+        elif name in ('indicators', 'tabulate'):
+            version = '.'.join(version.split('.')[:2])
```

Comment:
Could use use version numbers with two digits instead? It would be better not to have custom code for edge cases like this.

Reply:
Thanks for the note, updated :)